<launch>
	<node pkg="predictor" type="predictor" name="kalman_predictor" output="screen"/>
	<!-- <node pkg="rosbag"   	type="record"		 name="rosbag_record_prediction_ref" 
		args="record -O subset /target_pose_measurement /predicted_poses" /> -->
		<!-- rosbag record -O subset /target_pose_measurement /predicted_poses -->
</launch>